<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::PhoneInterpreterView</class>
 <widget class="QWidget" name="Qpid::PhoneInterpreterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voice Easy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#labelSourceLanguage,#labelTargetLanguage {
	font-weight: bold;
}

#frameMain QComboBox {
	border: 1px solid #cccccc;
	padding: 0px 18px 0px 3px;
	min-height: 30px;
}

#frameMain QComboBox:on { /* shift the text when the popup opens */
	padding-top: 3px;
	padding-left: 4px;
}

#frameMain QComboBox::drop-down {
	border: 0px;
	width: 39px;
}

#frameMain QComboBox::down-arrow {
	image: url(:/images/ui/language_down_arrow.png);
}

#frameCallContainer #pushButtonCall {
	color: #ffffff;
	font-weight: bold;
	font-family: Segoe UI;
	font-size: 18px;
	min-width: 122px;
	min-height: 34px;
	border-image: url(:/images/ui/button_call.png);
}

#frameCallContainer #pushButtonCall:hover {
	border-image: url(:/images/ui/button_call_hover.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="Qpid::CallInterpreterHelp" name="widgetCallHelp" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>176</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>176</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameMiddle">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frameMain">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <property name="leftMargin">
             <number>30</number>
            </property>
            <property name="topMargin">
             <number>30</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelSourceLanguage">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>21</height>
               </size>
              </property>
              <property name="text">
               <string>Select source language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxSourceLanguage">
              <property name="maximumSize">
               <size>
                <width>275</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTargetLanguage">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>21</height>
               </size>
              </property>
              <property name="text">
               <string>Select target language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxTargetLanguage">
              <property name="maximumSize">
               <size>
                <width>275</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frameCallContainer">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>9</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButtonCall">
                 <property name="minimumSize">
                  <size>
                   <width>122</width>
                   <height>34</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>Call</string>
                 </property>
                 <property name="default">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>185</width>
                   <height>11</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>105</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qpid::CallInterpreterHelp</class>
   <extends>QWidget</extends>
   <header>callinterpreterhelp.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="phone.qrc"/>
 </resources>
 <connections/>
</ui>
